docs/ops/normalization/NormalizeL2_1.md

```diff
@@ -19,32 +34,52 @@
 * *eps_mode*
 
   * **Description**: Specifies how *eps* is combined with L2 value calculated before division.
-  * **Range of values**: `add`, `max`
+  * **Range of values**: `add` or `max`
   * **Type**: `string`
   * **Default value**: None
   * **Required**: *yes*
 
 **Inputs**
 
-* **1**: `data` - input tensor to be normalized. Type of elements is any floating point type. Required.
+* **1**: `data` - A tensor of type *T* and arbitrary shape. **Required.**
 
-* **2**: `axes` - scalar or 1D tensor with axis indices for the `data` input along which L2 reduction is calculated. Required.
+* **2**: `axes` - Axis indices of `data` input tensor, along which L2 reduction is calculated. A scalar or 1D tensor of unique elements and type *T_IND*. The range of elements is `[-r, r-1]`, where `r` is the rank of `data` input tensor. **Required.**
 
 **Outputs**
 
-* **1**: Tensor of the same shape and type as the `data` input and normalized slices defined by `axes` input.
-
-**Detailed Description**
-
-Each element in the output is the result of division of corresponding element from the `data` input tensor by the result of L2 reduction along dimensions specified by the `axes` input:
+* **1**: Normalized tensor of the same shape and type as the `data` input.
```

Comment:
Suggestion: As other normalization operations. As we define formula/function above, mention it. Types usually go before shape in description.

\* \*\*1\*\*: The result of \*NormalizeL2\* function applied to \`data\` input tensor. Normalized tensor of the same type and shape as the `data` input.

Reply:
Thank you, done.